### PR TITLE
#174 Loading state, I've added loading to the component,  and class n…

### PR DESCRIPTION
**Pull request by andrewBalekha**
Not sure how to cover it with test, could someone help me out?

### CODE REVIEW

File: src/Geosuggest.jsx [hunk 1]
```diff
@@ -376,7 +380,8 @@ class Geosuggest extends React.Component {
     const attributes = filterInputAttributes(this.props),
       classes = classnames(
         'geosuggest',
-        this.props.className
+        this.props.className,
+        {'geosuggest-loading': this.state.isLoading}
```
Comment (Caerostris) [inline src/Geosuggest.jsx]: Shouldn't it be `geosuggest--loading` in accordance with BEM class naming?

**Comment by Caerostris** [comment]
Testing this will require some changes to the `autoCompleteApi` stub, so that we can run a `should()` at the right time.
🏁

**Comment by ro-ka** [comment]
Very nice! A example implementation would be awesome — mind to add one?

**Comment by dimafeldman** [comment]
That's really great! just one note - correct me if i'm wrong, the loading class name is being added  to the input element, right? if so - it might be a better idea to put it on the container, as input elements do not support css pseudo selectors like before/after which are commonly used for such loading indicators.

**Comment by Caerostris** [comment]
Yup, you are absolutely right!
It's already being added to the entire `div` container though 👍

**Comment by Caerostris** [comment]
@andrewBalekha Do you have an example of a loading animation which you could add to the example included with geosuggest?

**Comment by ro-ka** [comment]
I'll merge this one but adding a demo loading would still be nice.

**Comment by ro-ka** [comment]
Released in `1.25.0`.  🎉

**Pull request merged**